internal/provider/schema_cloudflare_tunnel_config.go

```diff
@@ -0,0 +1,177 @@
+									Description: "Maximum number of idle keepalive connections between Tunnel and your origin. This does not restrict the total number of concurrent connections.",
+									Default:     100,
+								},
+								"keep_alive_timeout": {
+									Type:        schema.TypeString,
+									Optional:    true,
+									Description: "Timeout after which an idle keepalive connection can be discarded.",
+									Default:     "1m30s",
+								},
+								"http_host_header": {
+									Type:        schema.TypeString,
+									Optional:    true,
+									Description: "Sets the HTTP Host header on requests sent to the local service.",
+									Default:     "",
+								},
+								"origin_server_name": {
+									Type:        schema.TypeString,
+									Optional:    true,
+									Description: "Hostname that cloudflared should expect from your origin server certificate.",
+									Default:     "",
+								},
+								"ca_pool": {
+									Type:        schema.TypeString,
+									Optional:    true,
+									Description: "Path to the certificate authority (CA) for the certificate of your origin. This option should be used only if your certificate is not signed by Cloudflare.",
+									Default:     "",
+								},
+								"no_tls_verify": {
+									Type:        schema.TypeBool,
+									Optional:    true,
+									Description: "Disables TLS verification of the certificate presented by your origin. Will allow any certificate from the origin to be accepted.",
+									Default:     false,
+								},
+								"disable_chunked_encoding": {
+									Type:        schema.TypeBool,
+									Optional:    true,
+									Description: "Disables chunked transfer encoding. Useful if you are running a Web Server Gateway Interface (WSGI) server.",
+									Default:     false,
+								},
+								"bastion_mode": {
+									Type:        schema.TypeBool,
+									Optional:    true,
+									Description: "Runs as jump host",
+								},
+								"proxy_address": {
+									Type:        schema.TypeString,
+									Optional:    true,
+									Description: "cloudflared starts a proxy server to translate HTTP traffic into TCP when proxying, for example, SSH or RDP. This configures the listen address for that proxy.",
+									Default:     "127.0.0.1",
+								},
+								"proxy_port": {
+									Type:        schema.TypeInt,
+									Optional:    true,
+									Description: "cloudflared starts a proxy server to translate HTTP traffic into TCP when proxying, for example, SSH or RDP. This configures the listen port for that proxy. If set to zero, an unused port will randomly be chosen.",
+									Default:     0,
+								},
+								"proxy_type": {
+									Type:         schema.TypeString,
+									Optional:     true,
+									Description:  "cloudflared starts a proxy server to translate HTTP traffic into TCP when proxying, for example, SSH or RDP. This configures what type of proxy will be started. Valid options are:.",
```

Comment:
```suggestion
									Description:  fmt.Sprintf("cloudflared starts a proxy server to translate HTTP traffic into TCP when proxying, for example, SSH or RDP. This configures what type of proxy will be started. %s", renderAvailableDocumentationValuesStringSlice([]string{"", "socks"}),
```

you will also need to import `fmt`